daemon/algod/api/server/v2/dryrun.go

```diff
@@ -397,17 +397,22 @@ func doDryrunRequest(dr *DryrunRequest, response *generated.DryrunResponse) {
 		return
 	}
 	proto := config.Consensus[protocol.ConsensusVersion(dr.ProtocolVersion)]
```

Comment:
shouldn't protocol version be set by a caller?

Reply:
check `func (v2 *Handlers) TealDryrun(ctx echo.Context) error`:
``` golang
	var protocolVersion protocol.ConsensusVersion
	if dr.ProtocolVersion != "" {
		var ok bool
		_, ok = config.Consensus[protocol.ConsensusVersion(dr.ProtocolVersion)]
		if !ok {
			return badRequest(ctx, nil, "unsupported protocol version", v2.Log)
		}
		protocolVersion = protocol.ConsensusVersion(dr.ProtocolVersion)
	} else {
		protocolVersion = hdr.CurrentProtocol
	}
	dr.ProtocolVersion = string(protocolVersion)
```

Reply:
ok, I changed the line in dryrun_test.go to have protocolversion = ""